src/serac/numerics/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(numerics_sources
     odes.cpp
     )
 
-set(numerics_depends serac_infrastructure)
+set(numerics_depends serac_infrastructure serac_functional)
```

Comment:
This is creating a circular dependency between numerics and functional

Reply:
Looking below it seems like that was already there implicitly through headers. That is unfortunate.

Reply:
Where is the circular dependency? 

I thought our directory structure was using the pattern where parent directories depended on the targets defined in their subdirectories (e.g. `serac` depends on `serac/numerics`, which depends on `serac/numerics/functional`)

Reply:
https://github.com/LLNL/serac/blob/9df86949f20f34fad863d7f1df054c5551c65d06/src/serac/numerics/functional/CMakeLists.txt#L7

Reply:
I agree based on directory structure what you added was "correct" but the existing one I just linked is the incorrect one.

Reply:
it looks like the quadrature data is the only piece being used from `numerics`, so it just should be moved

Reply:
Thank you for taking care of this. It is something that is easy to sneak in.